spp_area/models/area_import.py

```diff
@@ -212,6 +212,15 @@ def import_data(self):  # noqa: C901
                                 lang_ids.append([0, 0, lang[x]])
 
                             # Store values to columns
+                            char_type = "Int"
+                            for char in str(admin_code):
+                                if char.isalpha():
+                                    char_type = "Char"
+                                    break
+
+                            if char_type == "Int":
+                                admin_code = int(admin_code)
+
```

Comment:
```suggestion
                            char_type = "Int" and admin_code.isnumeric() or "Char"
                            if char_type == "Int":
                                admin_code = int(admin_code)

```

Reply:
@emjay0921 I second this so that we avoid the unnecessary for loop

Reply:
@emjay0921 sorry, I don't even know this column is float in your test excel file 😂, but this is something that should be improve in this code, huh ? I did not change the logic, just made it better formatted